src/util/irep_ids.def

```diff
@@ -330,6 +330,7 @@ IREP_ID_ONE(builtin_offsetof)
 IREP_ID_ONE(0)
 IREP_ID_ONE(1)
 IREP_ID_ONE(sizeof)
+IREP_ID_ONE(checked_assigns)
```

Comment:
Nit pick: any particular reason for using this line? I'd suggest to just add it at the very end.